FeedStoreChallenge/Model/ManagedFeedImage.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import CoreData
+
+class ManagedFeedImage: NSManagedObject {
```

Comment:
`ManagedFeedImage` is not supposed to be subclassed, so it should be marked as `final`:
```suggestion
final class ManagedFeedImage: NSManagedObject {
```